<launch>
    <arg name="open_rviz" default="false" />
    <arg name="open_reconfigure" default="false" />
    <arg name="rosplane_tuner" default="false" />

    <node name="xplane_wrapper" pkg="xplane_ros" type="xplane_ros_wrapper.py" output="screen" />
    <node name="ned_to_viz" pkg="xplane_ros" type="ned_to_viz.py" />
    <node name="xp_rosplane_viz" pkg="xplane_ros" type="rosplane_path_viz.py" />
    <node name="ai_commands" pkg="xplane_ros" type="ai_commands.py" output="screen" />
    <node name="traffic_sender" pkg="xplane_ros" type="traffic_sender.py" output="screen" />
    <node name="command_sender" pkg="xplane_ros" type="command_sender.py" output="screen" />


    <group if="$(arg open_rviz)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find xplane_ros)/rviz/default.rviz" />
    </group>
    <!-- <node name="ned_to_viz" pkg="tf" type="static_transform_publisher" args="0 0 0 3.14 0 0 world map 100" /> -->

        <include file="$(find rosplane_sim)/launch/ai_episode.launch" />
    <!-- <node name="xplane_takeoff" pkg="xplane_ros" type="takeoff.py" output="screen" /> -->


</launch>